packages/react-kitchen-sink/src/auth/useAuth.ts

```
@@ -2,7 +2,7 @@ import { captureException, setUser as setSentryUser } from '@sentry/react'
 import { authState } from '@valian/rxjs-firebase'
 import { type initializeApp } from 'firebase/app'
 import { getAuth, type User } from 'firebase/auth'
-import { catchError, map, NEVER, startWith } from 'rxjs'
+import { catchError, map, NEVER, startWith, tap } from 'rxjs'
```

ESLint

[error] 'tap' is defined but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)